server/www/templates/posts/post-editor.html

```diff
@@ -297,4 +297,13 @@ <h1 translate translate-values="{ type: activeForm.name, title: post.title }" ng
             </p>
         </div>
     </div>
+
+    <modal visible="userSavedPost">
```

Comment:
.. this should really be a confirmation message as per the pattern: http://github.ushahidi.org/platform-pattern-library/assets/html/4_blocks/#pl-pattern-confirmation

.. but this is better than alerts for launch